.github/pull_request_template.md

```diff
@@ -0,0 +1,9 @@
+# Changes
+- Change summary/description
+- Any @mention 's
+
+# Related Tickets
```

Comment:
Should we rename this as "Related Issues" since I don't think we're using tickets in Launch Pad? Please correct me if I'm wrong @johnagl

Reply:
I copied this from the launchpad doc repo's PR template and they were using `tickets` as the title but actually using `issues`. I thought they are the same but I not sure. @johnagl

Reply:
Issues is probably a bit clearer, but up to you if you want to change @danielchen-pyc :)